StayGo.Domain/Models/Bar.cs

```diff
@@ -0,0 +1,13 @@
+namespace StayGo.Domain.Models
+{
+    public class Bar
+    {
+        private int Id { get; set; }
```

Comment:
No need to make accessors private. How would you modify this class or even create if all accessors are private ?

Reply:
Understand, fix